rules/mincontains-zero-requires-maxcontains-zero.json

```diff
@@ -0,0 +1,65 @@
+{
+  "title": "`minContains: 0` requires `maxContains: 0`",
+  "description": "In draft-2019-09 and draft-2020-12, using `minContains:0` expresses that *no* array element may match `contains`. For clarity, also set `maxContains` to 0.",
```

Comment:
This isn't correct. `minContains: 0` means there can be no items that match the schema. An example is asserting that an array can contain at most one number.

```json
{
  "type": "array",
  "contains": { "type": "number" }
  "minContains": 0,
  "maxContains": 1
}
```

- PASS -- `["a", b", "c"]` // No matches
- PASS -- `["a", 2, "c"]` // One matches
- FAIL -- `["a", 2, 3]` // Two matches

Some rules that would make more sense
- `minContains: 0` without `maxContains`. Would never fail.
- `minContains > `maxContains`. Would never pass.
- `minContains: 0, maxContains: 0`. Same as `items: not: *schema*`.